lib/native-client.js

```diff
@@ -118,94 +118,6 @@ class NativeClient extends EventEmitter {
   }
 
   /**
-   * Execute a buildInfo command on the currently connected instance.
-   *
-   * @param {Function} callback - The callback.
-   */
-  buildInfo(callback) {
```

Comment:
This method is publically available in DataService. Will need to verify Compass is not using it and remove it from the public API.